app/services/payment_providers/adyen_service.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module PaymentProviders
+  class AdyenService < BaseService
+    def create_or_update(**args)
+      adyen_provider = PaymentProviders::AdyenProvider.find_or_initialize_by(
+        organization_id: args[:organization].id
```

Comment:
Style/TrailingCommaInArguments: Put a comma after the last parameter of a multiline method call. (https://rubystyle.guide#no-trailing-params-comma)